packages/mdc-drawer/index.js

```diff
@@ -15,5 +15,7 @@
  */
 
 import * as util from './util';
+export {MDCSlidableDrawerFoundation} from './slidable';
```

Comment:
Not sure if we want to export this particular foundation here since it's meant to be subclassed by our drawer implementations, and we want to reduce the API surface such that we're not encouraging users to make their own versions of Material Design drawers.